### ISSUE TITLE
individual means of transport for each favourite trip

### ISSUE THREAD

**Issue by francescosoave** [grote/Transportr#635]
**Acknowledgments**
Please check the following boxes with an `x` if they apply:
* [x]  The feature I want to propose would be useful for the majority of users, not only for me personally.
* [x] I am aware that Transportr is mostly developed by one person in their unpaid spare time.
* [] I can help myself to get this feature implemented or know someone who wants to do it.
* [x] If I want to add support for a new region or country, I checked that this is already available in [public-transport-enabler](https://github.com/schildbach/public-transport-enabler/) and know the process [described on the Transportr homepage](https://transportr.app/contribute/#translating-into-your-language).

**Is your feature request related to a problem? Please describe.**
I think it would be great if the means of transport could be saved for each favorite trip (as well as the possibility of saving a general means of transport settings for the app)

For example:
I take the bus in the morning. I use the app to check which bus number arrives first at my bus stop. I don't really want to see solutions that use trains etc but right now I have to edit that manually every time. On the other hand, on my way to the gym I need a bus + a subway so I need to manually select only those 2.
Both these trips are saved as favorite but I have to manually check/uncheck the means of transport every time for each of them.

Thank you.

**Describe the solution you'd like**
I think it would be great if each favorite trip could also have its own mean of transport preference.

**Describe alternatives you've considered**
No alternatives except to at least be able to save general preferences for the menas of transport (unless you consider this another feature request). In this case the preference would be applied to the whole app, for any search I'll do.

**Additional context**

**Comment by grote** [comment]
One would need to think about how to display and distinguish two trips with same start and end location, but different means of transport.

**Comment by francescosoave** [comment]
@grote good point, I didn't think about it. To be honest I'm not sure how many people would use the same trip but with different means of transport..also in that case I don't think the number of trips with same start/end but different means of transport would be very high so it might get away with it by just displaying a number (eg 1/2) next to the name of the trip. A user could easily remember that 1 is bus only and 2 is also with train for instance (this wouldn't work with many trips I guess)

**Comment by kajdo** [comment]
Hi,

I would love to see this feature ... I switchet to transportr from the app provided by my local transportation because, I'd happily replace everything which remotely fulfills my needs with opensource alternatives.

The features I wouldn't like to miss are addressed:
- create a desctop shortcut out of an favorite
- create favorites

The feature I'm missing is described in this issue. ... In my case - I'm living accross a main trainstation, so there are plenty of public transportation options. But for my daily trip to office it makes a huge difference in travel time (4min by train vs. >20min via bus/subway/....) - so I would love to see the option to also persist the choice of transportation within the favorite.

@grote if I got your feedback correctly - the different display could look like this ... not an designer - but I think that could do the trick
![image](https://user-images.githubusercontent.com/8735942/158819302-694d6a9a-fe4a-4f35-92d9-c48ca79b3ed7.png)

for me it would be great, to have a shortcut with only n specific means of transportation - and then create a shortcut for the launcher ... in the morning while drinking coffee - just 1 click and I know when to leave for the train :)
